testing/Readme.md

```diff
@@ -13,29 +13,30 @@ Using the Azure CLI, it creates the following:
-| 2     | Select PowerShell.                                                                                                                                                       | ![image](/docs/imgs/testing-secreenshots/shell2.png)  |
-| 3     | Upload `SetupTestbed.ps1` by clicking the "Upload/Download files" icon                                                                                                   | ![image](/docs/imgs/testing-screenshots/shell3.png)   |
-| 4     | Run the script, providing values for the parameters when promoted (see [Usage](#usage)). The script will take ~20 minutes to run to completion.                          | ![image](/docs/imgs/testing-screenshots/shell4.png)  |
-| 5     | Save the login credentials printed to the terminal at the end. At this point you can login to each VM using RDP (for the Windows servers) or SSH (for the Linux server). | ![image](/docs/imgs/testing-screenshots/shell5.png)                                           |
-| 6     | When you're done testing, simply delete the resource group to clean up all resources created.                                                                            | ![image](/docs/imgs/testing-screenshots/delete.png)                                           |
+| **#** | **Step**                                                                                                                                                                                                                                                              | **Screenshot**                                          |
+|-------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|---------------------------------------------------------|
+| 1     | Open a cloud shell by navigating to portal.azure.com and clicking the shell icon.                                                                                                                                                                                     | ![image](/docs/imgs/testing-screenshots/shell.png)      |
+| 2     | Select PowerShell.                                                                                                                                                                                                                                                    | ![image](/docs/imgs/testing-secreenshots/shell2.png)    |
+| 3     | Upload `SetupTestbed.ps1` by clicking the "Upload/Download files" icon  *Note: if you want to use the InstallTestbed script, you should probably clone the repo as described in that section, and run SetupTestbed.ps1 from the cloned repo in the testing directory* | ![image](/docs/imgs/testing-screenshots/shell3.png)     |
```

Comment:
Actually, you should _definitely_ clone the repo. Just uploading SetupTestbed.ps1 will not work anymore, even if you don't intend to run InstallTestbed.ps1.

Reply:
Done